### DISCUSSION TITLE
Can LL-HLS support MPEG-TS container format.

### DISCUSSION THREAD

**Discussion by xiaozhihong** [OvenMediaLabs/OvenMediaEngine#1286] [Help]
Can LL-HLS support MPEG-TS container format?

**Comment by getroot** [comment]
MPEG2-TS container is not supported because LLHLS requires the use of CMAF.

**Reply by xiaozhihong** [reply]
Can not find any description in [RFC8216](https://datatracker.ietf.org/doc/html/draft-pantos-hls-rfc8216bis) about this.

**Reply by getroot** [reply]
It seems that according to the specifications, Partial Segments can be provided in the MPEG2-TS container format. Personally, I haven't really considered providing Partial Segments in a format other than Fragmented MPEG-4. I thought Fragmented MPEG-4 was developed for Partial Segments. Also, since all the examples in the specifications are based on fMP4, it further reinforced that perception. Moreover, I'm not sure if all players would support Partial Segments in the MPEG2-TS format. Additionally, since HEVC is only supported in fMP4, I didn't initially consider MPEG2-TS support from the beginning.

**Reply by xiaozhihong** [reply]
Yes, I agree with you, thanks for your reply. And I implemented LL-HLS with MPEG-TS partial segment in my code, and it works well.

**Reply by getroot** [reply]
Thanks for good information
